neo/public/stylesheets/cloudberry.css

```diff
@@ -0,0 +1,77 @@
+/*
+* multi-line comment
```

Comment:
hmm. good point, I think it will eventually useful in the future for a better web console. I will clean it a little though.